### PR TITLE
Add new line after body header if attach present

### PR DESCRIPTION
**Pull request by no1seman**
Add absent new line between "content_type" and "body" if attachment present.

### CODE REVIEW

File: smtp/init.lua [hunk 1]
```diff
@@ -178,10 +178,10 @@ curl_mt = {
                 header = header .. table.concat(opts.headers, '\r\n') .. '\r\n'
             end
             local content_type = 'Content-Type: ' .. (opts.content_type or 'text/plain') ..
-            '; charset=' .. (opts.charset or  'UTF-8') ..';\r\n'
+            '; charset=' .. (opts.charset or  'UTF-8') ..';\r\n\r\n'
 
             if not opts.attachments or #opts.attachments == 0 then
-                body = header .. content_type ..'\r\n' .. body
+                body = header .. content_type .. body
```
Comment (Totktonada) [inline smtp/init.lua]: I would add extra `'\r\n'` between `content_type` and `body` in the if-branch that handles attachments (around lines 219-220) to improve code durability. Now it is easy to miss that content-type should always going right before body. The patch to illustrate the idea:

```diff
diff --git a/smtp/init.lua b/smtp/init.lua
index 47c813d..94ae12a 100644
--- a/smtp/init.lua
+++ b/smtp/init.lua
@@ -238,6 +238,7 @@ curl_mt = {
                        header ..
                        MULTIPART_SEPARATOR ..
                        content_type ..
+                       '\r\n' ..
                        body ..
                        attachments ..
                        MULTIPART_END
```

Comment (no1seman) [reply smtp/init.lua]: No, I disagree, seems it's better to add '\r\n' once in content_type and new line wiil be added in both cases.

**Review by no1seman — COMMENTED** [review]
Ok, Fixed

**Review by Totktonada — APPROVED** [review]
Looks correct.

Do you test it somehow?

**Pull request closed without merging**